src/languages/fsharp.js

```diff
@@ -369,11 +376,8 @@ export default function(hljs) {
       },
       {
         // computation expressions:
-        begin: [
-            /\b[_a-z]\w*/,
-            /(?=\s*\{)/
-          ],
-        beginScope: { 1: 'keyword' }
+        beginScope: 'computation-expression',
```

Comment:
Just `scope` should also work and reads a bit nicer.

Reply:
Oops, sorry!